tinyms/vision/transforms.py

```diff
@@ -611,8 +612,32 @@ def apply_ds(self, gan_generator_ds, repeat_size=1, batch_size=1,
         return gan_generator_ds
 
 
+
+class BertDatasetTransform():
```

Comment:
I think this class should be placed under `text` module, you can create a new `transform.py` file under text module.

Reply:
Thanks.I have add it to tinyms/text/transformers.py

Reply:
Thanks. I have add it to transforms.py.